plugins/woocommerce/client/blocks/assets/js/blocks/product-collection/edit/product-collection-content.tsx

```diff
@@ -155,7 +155,7 @@ const ProductCollectionContent = ( {
 	}
 
 	return (
-		<div { ...blockProps }>
```

Comment:
@gigitux  this one is needed. After choosing a collection (initial prompt), this is the main wrapper of the Product Collection. Without it Product Collection block is not highlighted in editor 

https://github.com/user-attachments/assets/3b61cef0-7f7d-4e44-a5a0-3e2ac7747a1c